public/index.html

```diff
@@ -9,11 +9,11 @@
     <!-- Lato regular, regular italic -->
     <link href="https://fonts.googleapis.com/css2?family=Lato:ital@0;1&display=swap" rel="stylesheet"> 
 
-    <title><%= htmlWebpackPlugin.options.title %></title>
+    <title>UBC Launchpad</title>
   </head>
   <body>
     <noscript>
-      <strong>We're sorry but <%= htmlWebpackPlugin.options.title %> doesn't work properly without JavaScript enabled. Please enable it to continue.</strong>
+      <strong>We're sorry but UBC Launchpad doesn't work properly without JavaScript enabled. Please enable it to continue.</strong>
```

Comment:
it might come across more naturally as:

```suggestion
      <strong>We're sorry but our website doesn't work properly without JavaScript enabled. Please enable it to continue.</strong>
```